ui/src/testing/react-test-helpers.tsx

```diff
@@ -190,5 +190,9 @@ export const expectDropdownDisabled = (
   ).not.toBeInTheDocument();
 };
 
+export const getHTMLInputElementValue = (element: HTMLElement): string => {
```

Comment:
Added helper function in order to avoid lots of casting in converted test.

Reply:
https://docs.google.com/document/d/1LDxH33OzrWM0-ZpgNrR5WQb8yG5AmaKqd7cZlLTo1v0/edit#heading=h.rp1m4x1tl8lb